pkg/cli/admin/inspect/inspect.go

```diff
@@ -117,6 +118,7 @@ func NewCmdInspect(streams genericiooptions.IOStreams) *cobra.Command {
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
 	cmd.Flags().StringVar(&o.sinceTime, "since-time", o.sinceTime, "Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time / since may be used.")
 	cmd.Flags().DurationVar(&o.since, "since", o.since, "Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of since-time / since may be used.")
+	cmd.Flags().StringVar(&o.nodeSelector, "node-selector", o.nodeSelector, "Set a specific node selector to use - only relevant for the daemonset pods. If the cluster's node count is less than or equal to 20, default is empty. If the cluster's node count is greater than 20, default is set to 'node-role.kubernetes.io/control-plane'")
```

Comment:
Can you set the default value in `NewInspectOptions` as part of `return &InspectOptions{`?

Reply:
Default value is dynamic based on the node count. Since there is no kubeconfig in `NewInspectOptions` (because initializations happen in `Complete` func), we show default as empty which is the default of "node count is less than threshold".

Reply:
After thinking on this, I'll apply the suggested change.

Reply:
Updated.